### DISCUSSION TITLE
How can I make my camera/view fly?

### DISCUSSION THREAD

**Discussion by lectsuliv** [SimDynamX/SpaceTeamsPRO#20] [General]
On the STUniversity Mars refrencesimulation I would like to make my camera/view fly or float, to allow for more visibility.  Is this possible? How can I make this happen? What part of the code do in json file do I need to change to do this?

**Comment by connorjak** [comment]
You can replace the pawn Entity (a `UIP2DSurf_Spectator` pawn):
![image](https://github.com/SimDynamX/SpaceTeamsPro/assets/19719887/bd34e9a7-8b96-427b-bdbe-d0fa47f021f6)

With a `UIP2D_Spectator` pawn, for example the one from `BaseSims/SetupNewSimStartingPoint.json`:
![image](https://github.com/SimDynamX/SpaceTeamsPro/assets/19719887/d12a5947-b34e-4366-a77a-f61bff30110f)

Make sure to copy over the Location and Dynamics/ResidentFrame so it starts in the same place!

**Comment by connorjak** [comment]
In the future, we will have tools for setting up a choice of pawn in the Sim Config Editor.
